packages/grafana-data/src/utils/logs.ts

```diff
@@ -1,6 +1,6 @@
 import { countBy, chain, escapeRegExp } from 'lodash';
```

Comment:
I have moved `logs sorting functions` from grafana `core` to `grafana/data` so we can use it in `grafana/ui` components such as `LogsRow`.